docs/launching-apps/lsf.rst

```diff
@@ -36,29 +36,23 @@ Launching
 When properly configured, Open MPI obtains both the list of hosts and
 how many processes to start on each host from LSF directly.  Hence, it
 is unnecessary to specify the ``--hostfile``, ``--host``, or ``-n``
-options to ``mpirun``.  Open MPI will use PBS/Torque-native mechanisms
+options to ``mpirun``.  Open MPI will use LSF-native mechanisms
 to launch and kill processes (``ssh`` is not required).
 
 For example:
 
-.. error:: TODO Need LSF specific content here
-
 .. code-block:: sh
 
-   # Allocate a PBS job with 4 nodes
-   shell$ qsub -I -lnodes=4
+   # Allocate a job with 4 nodes and run the job on LSF allocated nodes
+   shell$ bsub -n 4 "mpirun mpi-hello-world"
```

Comment:
The `-n` option allocates slots (often cores), not whole nodes. LSF syntax for getting a whole node is a little funny. I'd suggest changing it to something like the following:
```suggestion
   # Allocate a job using 4 nodes with 2 processors per node and run the job on the nodes allocated by LSF
   shell$ bsub -n 8 -R "span[ptile=2]" "mpirun mpi-hello-world"
```

Reply:
I agree, and thought about this `ptile` addition, however the other schedule examples did not introduce any *complicated* flags. But by all means I am pro having this explicitly if needed.

Reply:
Unfortunately, LSF (except in the CORAL/CSM configuration) doesn't have an easy option to get a full node (that I know of anyway. So the user needs to add more complicated options and know the node sizes ahead of time. Since asking for a whole node is pretty common I think it is good to put this more complex option for users to reference.